README.md

```diff
@@ -17,26 +17,23 @@ Vets-api requires:
 1. Install Postgres (on Mac): `brew install postgres`
 1. Get the code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git; git submodule init; git submodule update`
 1. Install gem dependencies: `cd vets-api; bundle install`
+1. Create a application.yml `cat ./application.yml.example > ./application.yml`
```

Comment:
Just started going through this--there's no application.yml.example in this directory--did you mean to incluce it in this PR?